Common/src/main/java/dk/s4_g1/common/data/StatusCode.java

```diff
@@ -0,0 +1,9 @@
+package dk.s4_g1.common.data;
+
+public class StatusCode {
+    public final int statusCode;
+
+    public StatusCode(int statusCode){
+        this.statusCode = statusCode;
+    }
+}
```

Comment:
Consider changing this to an `enum` with all the different responses.

Reply:
How do you think that enum should be implemented? 
something like
```java
public enum Status {
    OK(),
    ERROR(),
    NOTFOUND(),
}

public class StatusCode {
    public final int statusCode;
    public final Status status;

    public StatusCode(int statusCode, Status status){
        this.statusCode = statusCode;
        this.status = status;
    }
}
```